.golangci.yml

```diff
@@ -23,11 +23,12 @@ linters-settings:
   golint:
     min-confidence: 0.8
   govet:
+    enable:
+      - fieldalignment
     check-shadowing: true
+
   lll:
     line-length: 140
-  maligned:
-    suggest-new: true
```

Comment:
Is there something like `suggest-new: true` for govet? IMO it is hard to fix this lint errors without linter's suggestion.

Reply:
Sadly there isn't any and fixing all of those stuff is not easy. Not to mention the struct readability goes away with this. I checked several projects and for now -- until the maintainers provide a better way to handle these pointer byte errors -- they have it disabled.

Reply:
yo, just found an easy way to fix `fieldalignment` lint errors
sadly it isn't available in `golangci-lint` yet
```sh
go get golang.org/x/tools/go/analysis/passes/fieldalignment/cmd/fieldalignment
fieldalignment -fix ./...
```